### PR TITLE
feat: #10 extend PopulationRecommendation profile

### LINKED ISSUE #10: Model PopulationRecommendation

### PR DESCRIPTION
**Pull request by n-krueger**
Closes #10

### CODE REVIEW

File: ig/input/fsh/PopulationRecommendation.fsh [hunk 1]
```diff
@@ -12,13 +13,45 @@ Description: ""
 * extension contains
   ageStart 0..1 MS and
   ageEnd 0..1 MS
-* extension[ageStart].value[x] only Quantity
-* extension[ageStart].value[x] obeys age-1
-* extension[ageStart].value[x].code from http://hl7.org/fhir/ValueSet/all-time-units
+* extension contains VPPopulationRecommendationLocationExtension named location 0..* MS
+
+* extension[ageStart]
```
Comment (tklausen) [inline ig/input/fsh/PopulationRecommendation.fsh:31]: ageStart and ageEnd should be the same as in VaccinationScheme.

File: ig/input/fsh/PopulationRecommendation.fsh [hunk 2]
```diff
@@ -3,7 +3,8 @@ Parent: Basic
 Id: vp-population-recommendation
 Title: "PopulationRecommendation"
 Description: ""
```
Comment (tklausen) [inline ig/input/fsh/PopulationRecommendation.fsh]: Can you please add a short description?

Comment (tklausen) [inline ig/input/fsh/PopulationRecommendation.fsh]: Can we add the MS flag to `code` so it shows up in the must-support snapshot table? It's already set to 1..1 by default.

File: ig/input/fsh/PopulationRecommendation.fsh [hunk 3]
```diff
@@ -12,13 +13,45 @@ Description: ""
 * extension contains
   ageStart 0..1 MS and
   ageEnd 0..1 MS
-* extension[ageStart].value[x] only Quantity
-* extension[ageStart].value[x] obeys age-1
-* extension[ageStart].value[x].code from http://hl7.org/fhir/ValueSet/all-time-units
+* extension contains VPPopulationRecommendationLocationExtension named location 0..* MS
```
Comment (tklausen) [inline ig/input/fsh/PopulationRecommendation.fsh:21]: Just to clarify:
A PopulationRecommendation that should be applied independent of any location doesn't have a location, right?

**Review by tklausen — CHANGES_REQUESTED** [review]
Very nice work, I just have a few suggestions. Most of them are very similar to the VaccinationScheme PR (#27).

File: ig/input/fsh/PopulationRecommendation.fsh (hunk 3 above)
Comment (n-krueger) [reply ig/input/fsh/PopulationRecommendation.fsh:21]: Yes, that's correct. Likewise not providing `ageStart` or `ageEnd` implies that there is no start/end restriction on the age of the patient.

File: ig/input/fsh/PopulationRecommendation.fsh (hunk 2 above)
Comment (tklausen) [inline ig/input/fsh/PopulationRecommendation.fsh]: We forgot to model the Reference to TargetDisease.

File: ig/input/fsh/PopulationRecommendation.fsh (hunk 3 above)
Comment (tklausen) [reply ig/input/fsh/PopulationRecommendation.fsh:21]: Okay, great! Thanks!

File: ig/input/fsh/PopulationRecommendation.fsh (hunk 1 above)
Comment (n-krueger) [reply ig/input/fsh/PopulationRecommendation.fsh:31]: Addressed at da97313

File: ig/input/fsh/PopulationRecommendation.fsh (hunk 2 above)
Comment (n-krueger) [reply ig/input/fsh/PopulationRecommendation.fsh]: Addressed at da97313

Comment (n-krueger) [reply ig/input/fsh/PopulationRecommendation.fsh]: Addressed at da97313

Comment (n-krueger) [reply ig/input/fsh/PopulationRecommendation.fsh]: I have added a `targetDisease` extension (`1..1 MS`) with an ICD-10 code of the disease to immunise against.

Comment (tklausen) [reply ig/input/fsh/PopulationRecommendation.fsh]: Great!

**Review by tklausen — APPROVED** [review]
Good job!

**Pull request merged**